symphony-bdk-core/src/main/java/com/symphony/bdk/core/auth/impl/AppAuthSessionRsaImpl.java

```diff
@@ -69,4 +71,8 @@ public void refresh() throws AuthUnauthorizedException {
   protected ExtensionAppAuthenticator getAuthenticator() {
     return authenticator;
   }
+
+  public Object validateJwt(String jwt) throws AuthInitializationException {
+    return JwtHelper.validateJwt(jwt, authenticator.getPodCertificate().getCertificate());
```

Comment:
We should think about caching the pod certificate. Otherwise each jwt validation will cost a call to SBE

Reply:
Story opened to design and implement a caching mechanism: https://perzoinc.atlassian.net/browse/APP-3091